ee/tabby-webserver/src/service/license.rs

```diff
@@ -59,18 +60,40 @@ fn jwt_timestamp_to_utc(secs: i64) -> Result<DateTime<Utc>> {
 
 struct LicenseServiceImpl {
     db: DbConn,
+    seats: RwLock<(DateTime<Utc>, usize)>,
+}
+
+impl LicenseServiceImpl {
+    async fn seats(&self) -> Result<usize> {
+        let now = Utc::now();
+        let lock = self.seats.read().await;
+        let (refreshed, seats) = &*lock;
```

Comment:
always return `seats`.

But when cache expired, spawn a task to update seat in a background job